components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -70,10 +71,11 @@ export default function Preferences() {
                         );
                     }
                 }
+                // Reset creationError to avoid displaying the error message and toast the success message
```

Comment:
This would have been a good opportunity to refactor this into a react-query mutation, which handles the error state for us automatically.